packages/cadl-ranch-specs/http/models/property-nullable/mockapi.ts

```diff
@@ -0,0 +1,90 @@
+import { passOnSuccess, ScenarioMockApi, mockapi, json, MockApi } from "@azure-tools/cadl-ranch-api";
+
+export const Scenarios: Record<string, ScenarioMockApi> = {};
+
+interface MockApiGetPut {
+  getAll: MockApi;
+  getDefault: MockApi;
+  patchAll: MockApi;
+  patchDefault: MockApi;
+}
+
+/**
+ * Return the get and put operations
+ * @param route The route within /models/properties/optional/all/ for your function.
+ * @param value The value you are expecting and will return
+ */
+function createMockApis(route: string, value: any): MockApiGetPut {
+  const url = `/models/properties/nullable/${route}`;
+  const allUrl = `${url}/all`;
+  const defaultUrl = `${url}/default`;
```

Comment:
Would just say here that `default` doesn't make that much sense for nullable as null is a value and we are not involving the default values.

Maybe name `withNull` and `withoutNull` or anything else you feel is more descriptive of the fact that we are testing sending null or not.